test/functional/webauthn_verifications_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require "test_helper"
+
+class WebauthnVerificationsControllerTest < ActionController::TestCase
+  context "#verify" do
+    context "when given an expired webauthn token" do
+      setup do
+        @user = create(:user)
+        token = create(:webauthn_verification, user: @user, path_token_expires_at: 1.minute.ago).path_token
+        get :verify, params: { webauthn_token: token }
+      end
+
+      should "return a 404" do
+        assert_response :not_found
+      end
+    end
+
+    context "when given an invalid webauthn token" do
+      setup do
+        @user = create(:user)
+        get :verify, params: { webauthn_token: "not_valid1234" }
+      end
+
+      should "return a 404" do
+        assert_response :not_found
+      end
+    end
+
+    context "when given a valid webauthn token param" do
+      setup do
+        @user = create(:user)
+        create(:webauthn_credential, user: @user)
+        token = create(:webauthn_verification, user: @user).path_token
```

Comment:
Nit: this is a verification, not a token.

Reply:
Same as above, here I'm calling `verification.path_token`